src/main/kotlin/uk/gov/justice/digital/assessments/jpa/entities/ReferenceDataTargetMappingEntity.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.assessments.jpa.entities
+
+import javax.persistence.*
```

Comment:
Not sure why wildcard imports aren't being picked up by the linter!

Reply:
Maybe I'm missing a trick here? - I've updated the threshold on Intellij so this hopefully doesn't happen in the future